spec/webauthn/signature_verifier_spec.rb

```diff
@@ -63,7 +63,7 @@
   end
 
   context "PS256" do
-    before do
+    before(:all) do
```

Comment:
Maybe just me, but always found `before(:all)` kind of confusing and a bit hacky...

Would it be more obvious and explicit to read if we have the `skip` call next to the `sign_pss` call?

```diff
diff --git a/spec/webauthn/attestation_statement/tpm_spec.rb b/spec/webauthn/attestation_statement/tpm_spec.rb
index 6e2fe95..8c2dadb 100644
--- a/spec/webauthn/attestation_statement/tpm_spec.rb
+++ b/spec/webauthn/attestation_statement/tpm_spec.rb
@@ -270,15 +270,13 @@ RSpec.describe "TPM attestation statement" do
       end
 
       context "when RSA PSS algorithm" do
-        before(:all) do
-          unless OpenSSL::PKey::RSA.instance_methods.include?(:verify_pss)
-            skip "Ruby OpenSSL gem #{OpenSSL::VERSION} do not support RSASSA-PSS"
-          end
-        end
-
         let(:algorithm) { -37 }
         let(:signature) do
-          aik.sign_pss("SHA256", to_be_signed, salt_length: :max, mgf1_hash: "SHA256")
+          if OpenSSL::PKey::RSA.instance_methods.include?(:verify_pss)
+            aik.sign_pss("SHA256", to_be_signed, salt_length: :max, mgf1_hash: "SHA256")
+          else
+            skip "Ruby OpenSSL gem #{OpenSSL::VERSION} do not support RSASSA-PSS"
+          end
         end
 
         it "works if everything's fine" do
```
?